ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV4.java

```diff
@@ -55,8 +55,12 @@ protected ValidationResult<RpcErrorType> validateParameters(
       final EnginePayloadParameter payloadParameter,
       final Optional<List<String>> maybeVersionedHashParam,
       final Optional<String> maybeBeaconBlockRootParam) {
-    if (payloadParameter.getBlobGasUsed() == null || payloadParameter.getExcessBlobGas() == null) {
-      return ValidationResult.invalid(RpcErrorType.INVALID_PARAMS, "Missing blob gas fields");
+    if (payloadParameter.getBlobGasUsed() == null) {
+      return ValidationResult.invalid(
+          RpcErrorType.INVALID_BLOB_GAS_USED_PARAMS, "Missing blob gas used field");
+    } else if (payloadParameter.getExcessBlobGas() == null) {
+      return ValidationResult.invalid(
+          RpcErrorType.INVALID_PARAMS, "non-null ExcessBlobGas pre-cancun");
```

Comment:
There is, but it will be coming in a later PR as I'm updating them in alphabetical order to ensure I don't miss any. This is the 7th PR in the series, breaking apart https://github.com/hyperledger/besu/pull/7389 which is too gigantic to review.